client/post-editor/restore-post-dialog.jsx

```diff
@@ -50,7 +50,7 @@ export default React.createClass( {
 				key="back"
 				isPrimary={ false }
 				onClick={ this.props.onClose }>
-					{ this.translate( 'Close' ) }
+					{ this.translate( 'Don\'t Restore' ) }
```

Comment:
Use double quotes here to avoid escaping is valid in eslint if the string contains a single quote:
```js
{ this.translate( "Don't Restore" ) }
```